src/modules/elements/time-picker/element.js

```diff
@@ -184,18 +187,18 @@ TimePicker.propTypes = {
 	 * using 24h clock in hh:mm format
 	 * e.g. 00:24, 03:57, 21:12
 	 */
-	current: TribePropTypes.timeFormat.isRequired,
-	min: TribePropTypes.timeFormat,
-	max: TribePropTypes.timeFormat,
-	start: TribePropTypes.timeFormat.isRequired,
+	allDay: PropTypes.bool,
+	current: TribePropTypes.string,
```

Comment:
Question here what's the difference with `PropTypes.string`

Reply:
We're changing the timepicker component from a time input to a text input. Because the format won't always be a time format, let alone a 24-hour time format, it needs to be a string.

Reply:
Do we have a `TribePropTypes.string` in common? I don't see it in `M18.19`

Reply:
you're right, this should be `PropTypes.string`